crates/rust-analyzer/src/handlers/request.rs

```diff
@@ -520,7 +520,10 @@ pub(crate) fn handle_workspace_symbol(
 
             #[allow(deprecated)]
             let info = SymbolInformation {
-                name: nav.name.to_string(),
+                name: match nav.alias {
+                    Some(ref alias) => format!("{} (alias {})", alias, nav.name),
```

Comment:
```suggestion
                name: match &nav.alias {
                    Some(alias) => format!("{} (alias {})", nav.name, alias),
```

This order matches the order for how we render them in completions

Reply:
I though about this as well. First of all your [comment](https://github.com/rust-lang/rust-analyzer/pull/14849#discussion_r1203762825)  above stated that I should use `<alias> ( alias <original_name> )` which I wanted to oppose in the first place but after looking at how VS Code's symbol search feature works I saw that if the alias doesn't come as the first element it is not possible to search for it as workspace symbols can only be searched for the longest matching prefix. So this change is not ideal. In order to further clarify maybe we could use `<alias> ( alias for/of <original> )` 

PS : While I was working on the first version I actually could use the format you suggest and it was working just fine if I remember correctly but somehow it is not working now. So maybe this is related to me using an older version of VSCode.

Reply:
> but after looking at how VS Code's symbol search feature works I saw that if the alias doesn't come as the first element it is not possible to search for it as workspace symbols can only be searched for the longest matching prefix.

Ah, I assumed it was doing fuzzy matching. Then it has to stay like this, unfortunately. Let's add a `for` to make it a bit clearer what's what.